### ISSUE TITLE
Write first complete methods section

### ISSUE THREAD

**Issue by svenvanderburg** [matchms/ms2deepscore#69]
- [x] spectrum data preparation
- [x] similarity labels preparation
- [x] training data generation
- [x] data augmentation
- [x] evaluation metrics (i.e. RMSE)
- [x] deep learning model + training

**Comment by svenvanderburg** [comment]
@florian-huber is this OK for a first complete section or am I missing something?

**Comment by florian-huber** [comment]
Thanks a lot @svenvanderburg. Looks good and will be mostly finetuning from here on. So let's close this issue!
